src/OrleansProviders/Streams/Memory/IMemoryStreamQueueGrain.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Memory
+{
+    public interface IMemoryStreamQueueGrain : IGrainWithGuidKey
+    {
+        Task SetMaxEventCount(int maxEventCount);
```

Comment:
This is supposed to be a helper function for users that limits the size of the queue. It doesn't have to be called during the provider's life cycle.